src/pages/Profile.js

```diff
@@ -279,20 +290,10 @@ query charts($geoCode: String!, $geoLevel: String!) {
               }
             >
               <ChartContainer
-                overflowX={
-                  chart.visuals.find(visual => visual.type === 'pie')
-                    ? 'visible'
-                    : chart.visuals.find(visual => visual.horizontal)
-                    ? 'hidden'
-                    : 'auto'
-                }
-                overflowY={
-                  chart.visuals.find(visual => visual.type === 'pie')
-                    ? 'visible'
-                    : chart.visuals.find(visual => visual.horizontal)
-                    ? 'auto'
-                    : 'hidden'
-                }
+                /* TODO: hurumap-ui remove scroll */
+                overflowX="visible"
+                overflowY="visible"
```

Comment:
Should these still be here?

Reply:
Am waiting for the new `hurumap-ui` to be published then all these will be gone.